src/main/python/pybuilder/plugins/python/pylint_plugin.py

```diff
@@ -44,4 +45,63 @@ def execute_pylint(project, logger):
     logger.info("Executing pylint on project sources")
 
     command_and_arguments = ["pylint"] + project.get_property("pylint_options")
-    execute_tool_on_modules(project, "pylint", command_and_arguments, True)
+    pylint_output_file_path = execute_tool_on_modules(project, "pylint", command_and_arguments, True)[1]
+
+    with open(pylint_output_file_path, 'r') as f:
+        file_content = f.read().splitlines()
+
+    module_name = ''
+    pylint_score = 0
+    pylint_change = 0
+    errors = 0
+    errors_info = ''
+    warnings = 0
+    warnings_info = ''
+    show_info_messages = project.get_property('pylint_show_info_messages')
+    show_warning_messages = project.get_property('pylint_show_warning_messages')
+    for line in file_content:
+        if line.startswith('************* Module'):
+            module_name = line.split(' ')[2]
+
+        if line.startswith('E:') or line.startswith('F:'):
+            logger.error('Pylint: Module %s: ' % module_name + line)
+            errors += 1
+
+        if show_warning_messages and line.startswith('W:'):
+            logger.warn('Pylint: Module %s: ' % module_name + line)
+            warnings += 1
+
+        if show_info_messages and (line.startswith('C:') or line.startswith('R:')):
+            logger.info('Pylint: Module %s: ' % module_name + line)
+
+        if line.startswith('Your code has been rated at '):
+            pylint_score = float(line.split(' ')[6].split('/')[0])
+            pylint_change = float(line.split(' ')[10][:-1])
+
+    if errors > 0:
+        errors_info = ' / Errors: {}'.format(errors)
+
+    if warnings > 0:
+        warnings_info = ' / Warnings {}'.format(warnings)
+
+    logger.info(
+        'Pylint ratio: {} / Pylint change: {}'.format(pylint_score, pylint_change) + errors_info + warnings_info
+    )
+
+    if errors > 0 and project.get_property('pylint_break_build_on_errors'):
```

Comment:
flake8 and frosted use `*_break_build` property. I suggest to do the same: use `pylint_break_build`

Reply:
Hi @AlexeySanko thanks for quick response to PR. In case of pylint we have a little different situation. In this plugin we assumed we can fail from different reasons:
- errors in output
- pylint score too low
- pylint change of score (for example from 9.0 to 8.5) is too big
This is why we used 'pylint_break_build_on_errors' for the first case.
Using 'pylint_break_build' suggests this is global brake/not brake which is not true in this case.
Do you agree with this approach? Do you have suggestion for better naming in this case?

Reply:
Hi, @maciejpolanczyk 
I thought that You will not support RP and created external plugin [`pybuilder_pylint_extended`](https://github.com/AlexeySanko/pybuilder_pylint_extended).
I used idea that plugin has to break build on errors and fatals anyway - otherwise You can disable and do not use plugin. Errors are errors and have to be fixed.

Reply:
Good point, we will change it to work as you described.